src/components/pages/NotFound/NotFound.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import style from './NotFound.module.scss';
+import { useNavigate } from 'react-router-dom';
+
+const NotFound: React.FC = () => {
+  const navigate = useNavigate();
+
+  const goBack = () => navigate(-1);
+
+  return (
+    <div className={style.body}>
+      <div className={style.items}>
+        <span className={style.item}>4</span>
+        <span className={style.item}>0</span>
+        <span className={style.item}>4</span>
+      </div>
+      <p className={style.text}>Oh, man. Page not found</p>
+      <button className="button" onClick={goBack}>
```

Comment:
In such case a variable is redundant. For more clear code:
```<button className="button" onClick={() => navigate(-1)}>```

Reply:
done